packages/flutter_tools/test/integration.shard/test_data/hot_reload_errors_common.dart

```diff
@@ -53,16 +54,36 @@ void testAll({
             ),
           ),
         );
+        await expectLater(flutter.hotRestart(), completes);
       },
     );
 
-    testWithoutContext('hot restart succeeds when removing a field from a const class', () async {
-      await flutter.run(
-        device: chrome ? GoogleChromeDevice.kChromeDeviceId : FlutterTesterDevices.kTesterDeviceId,
-        additionalCommandArgs: additionalCommandArgs,
-      );
-      project.removeFieldFromConstClass();
-      await expectLater(flutter.hotRestart(), completes);
-    });
+    testWithoutContext(
+      'Expression evaluation succeeds after a hot reload rejection error',
+      () async {
+        await flutter.run(
+          device:
+              chrome ? GoogleChromeDevice.kChromeDeviceId : FlutterTesterDevices.kTesterDeviceId,
+          withDebugger: true,
+          additionalCommandArgs: additionalCommandArgs,
+        );
+        project.removeFieldFromConstClass();
```

Comment:
Should we call `flutter.hotReload()` after this?

Reply:
Yeah I wonder if we should as well. The original error appeared after the rejection with an expression evaluation.

Reply:
Opps back copy/paste. Yes, the reload should be there. Added.